plugins/aea-ledger-ethereum-hwi/aea_ledger_ethereum_hwi/account.py

```diff
@@ -101,6 +100,21 @@ class UnsignedDynamicTransaction(HashableRLP):
     ]
 
 
+class UnsignedType1Transaction(HashableRLP):
+    """Unsigned typ1 transaction"""
+
+    fields = [
+        ("chainId", rlp.sedes.big_endian_int),
+        ("nonce", rlp.sedes.big_endian_int),
+        ("gasPrice", rlp.sedes.big_endian_int),
+        ("gas", rlp.sedes.big_endian_int),
+        ("to", address),
+        ("value", rlp.sedes.big_endian_int),
+        ("data", rlp.sedes.binary),
+        ("accessList", access_list_sede_type),
+    ]
```

Comment:
We will have to use type 1 legacy transactions since we include the chain id by default